services/QuillLMS/app/models/concerns/public_progress_reports.rb

```diff
@@ -150,7 +153,7 @@ def finished_activity_sessions_for_unit_activity_classroom_and_student(unit_id,
     classification = Activity.find_by(id: activity_id).classification
     student = User.find_by(id: student_id)
 
-    activity_sessions.map { |activity_session| formatted_score_obj(activity_session, classification, student) }
+    activity_sessions.map { |activity_session| formatted_score_obj(activity_session, activity.classification, student) }
```

Comment:
I'm confused about how this is doing anything different, considering how `classification` is defined in line 153.

Reply:
Good callout - I removed line 153, since it's now redundant